### DISCUSSION TITLE
Future features Cyclers-Controler

### DISCUSSION THREAD

**Discussion by pastorpflores** [WattRex/Regenerative-Battery-Cycler#108] [Ideas]
# Next ideas to improve design

**Comment by RobertoAldea** [comment]
In the epc driver the disable function actually sends a command to apply the wait mode for 10ms as the EPC doesn´t allow to send the epc directly to IDLE mode, so there is almost no difference with set wait mode.
In future versions an option could be to send a message to the epc to get the actual mode, and resend it back disabled
